sdk/eventgrid/eventgrid/package.json

```diff
@@ -34,8 +34,8 @@
   "//metadata": {
     "constantPaths": [
       {
-        "path": "src/generated/generatedClientContext.ts",
-        "prefix": "packageVersion"
+        "path": "swagger/README.md",
```

Comment:
oh interesting,  I don't think we've had the bot update the swagger README before, though it won't be able to re-run autorest as part of the version bump PR, so maybe add it to both places?

Reply:
Turns out there was nothing in `src/generated/generatedClientContext.ts` anymore that needed updating (perhaps due to the move to core-http2?  I will admit I did not chase it down).